src/web/components/Links.tsx

```diff
@@ -91,17 +91,19 @@ const seperatorHideStyles = css`
 `;
 
 const Search = ({
-	className,
 	children,
 	href,
 	dataLinkName,
 }: {
-	className?: string;
 	children: React.ReactNode;
 	href: string;
 	dataLinkName: string;
 }) => (
-	<a href={href} className={className} data-link-name={dataLinkName}>
+	<a
+		href={href}
+		css={[linkTablet({ showAtTablet: false }), searchLinkStyles]}
```

Comment:
before `className` was passed down as a prop. However the component `Search` is only ever used in this file once. So it makes sense to move the `css` directly to the component